src/main/java/net/sourceforge/plantuml/abel/LeafType.java

```diff
@@ -84,7 +87,7 @@ public boolean isLikeClass() {
 		return this == LeafType.ANNOTATION || this == LeafType.ABSTRACT_CLASS || this == LeafType.CLASS
 				|| this == LeafType.INTERFACE || this == LeafType.ENUM || this == LeafType.ENTITY
 				|| this == LeafType.PROTOCOL || this == LeafType.STRUCT || this == LeafType.EXCEPTION
-				|| this == LeafType.METACLASS || this == LeafType.STEREOTYPE;
+				|| this == LeafType.METACLASS || this == LeafType.STEREOTYPE || this == LeafType.DATA_CLASS || this == RECORD;
```

Comment:
Hi @Intybyte 

Isn't there an `LeafType` missing to make `LeafType.RECORD`?

Regards,
Th.